wsdl-cli/src/main/java/io/ballerina/wsdl/cli/WsdlCmd.java

```diff
@@ -107,37 +107,41 @@ public WsdlCmd() {
 
     @Override
     public void execute() {
-        if (helpFlag) {
+        if (this.helpFlag) {
             StringBuilder stringBuilder = new StringBuilder();
             printLongDesc(stringBuilder);
-            outStream.println(stringBuilder);
+            this.outStream.println(stringBuilder);
+            exitOnError();
             return;
         }
-        Path modulePath = getModulePath();
-        if (!isValidBallerinaProject()) return;
-        if (!isValidModuleName(moduleName)) return;
-        if (inputPath.isEmpty()) {
-            outStream.println(MISSING_WSDL_PATH);
+        if (this.inputPath == null || this.inputPath.isEmpty()) {
+            this.outStream.println("A WSDL file path is required to generate the client and types.");
+            this.outStream.println("e.g: bal wsdl <wsdl file>");
```

Comment:
```suggestion
            this.outStream.println("e.g: bal wsdl <wsdl-file>");
```
Something like this?